lib/cext/preprocess.rb

```diff
@@ -55,6 +55,8 @@ def preprocess(line)
 end
 
 if __FILE__ == $0
+  puts "# 1 \"#{ARGF.filename}\""
```

Comment:
I don't think `#` alone is a valid preprocessor directive, is it?
`#line` is definitely more standard and explicit at least.